homeassistant/helpers/recorder.py

```diff
@@ -4,12 +4,11 @@
 from homeassistant.core import HomeAssistant
 
 
-async def async_migration_in_progress(hass: HomeAssistant) -> bool:
+def async_migration_in_progress(hass: HomeAssistant) -> bool:
```

Comment:
Is this a breaking change for custom integrations?

Reply:
I don't think that get used outside of core

Reply:
Does not seem like it, me and @emontnemery did a scan of 592 custom integrations, none of them used it

Reply:
I've added a breaking change section for good measure, but I think it's OK to do the change.